docker/dbt/jaffle_shop/models/marts/core/schema.yml

```diff
@@ -32,6 +32,16 @@ models:
       - name: total_order_amount
         description: Total value (AUD) of a customer's orders
 
+  - name: dim_order_dates
+    description: This table is built in an incremental fashion and contains all the dates on which orders have been placed.
```

Comment:
I believe that there are some incremental IT tests to be enabled in `trino.dbtspec`.

Reply:
You're right.
I'm looking whether the adapter test for incremental models can be applied in this case.

Reply:
Thanks for adding this, it looks like the test is passing.